components/apps.js

```diff
@@ -592,47 +599,38 @@ SteamUser.prototype.getOwnedApps = function(excludeSharedLicenses) {
  * Check if this account owns an app. Only works if enablePicsCache option is enabled and appOwnershipCached event
  * has been emitted.
  * @param {int} appid
- * @param {boolean} [excludeSharedLicenses=false] - Pass true to exclude licenses that we have through family sharing
+ * @param {object} options - Options for what counts for ownership
  * @returns {boolean}
  */
-SteamUser.prototype.ownsApp = function(appid, excludeSharedLicenses) {
-	return this.getOwnedApps(excludeSharedLicenses).indexOf(parseInt(appid, 10)) != -1;
+SteamUser.prototype.ownsApp = function(appid, options) {
+	return this.getOwnedApps(options).indexOf(parseInt(appid, 10)) != -1;
```

Comment:
Propagate `options` to eventually `getOwnedPackages`. The same goes for  `getOwnedApps`, `ownsDepot`, `getOwnedDepots` and `ownsPackage`.